### PR TITLE
Metrics monitoring guide: update getting started content

### PR DESCRIPTION
**Pull request by EamonnTP**
This PR updates the [Installing metrics](https://www.elastic.co/guide/en/metrics/guide/current/install-beats-for-metrics.html) content to provide a better user experience to what is currently offered.

The content now includes step procedures:

1. Download and install Metricbeat.
1. Configure Metricbeat.
1. Start Metricbeat.
1. Verify your data in Kibana.

### HTML preview

https://observability-docs_58.docs-preview.app.elstc.co/guide/en/metrics/guide/master/install-metrics-monitoring.html

### Related issue

Closes https://github.com/elastic/observability-docs/issues/11

### CODE REVIEW

File: docs/en/metrics/metrics-installation.asciidoc [hunk 1]
```diff
@@ -1,100 +1,107 @@
...
+== Get started
 
-The easiest way to get started with Elastic Logs is by using our hosted {es} Service on Elastic Cloud.
-The {es} Service is available on both AWS and GCP, and automatically configures {es} and {kib}.
+To use the {metrics-app}, you need {es} for storing and searching your data, and {kib}
+(version 6.5 or later) for visualizing and managing it.
 
-NOTE: If your data uses nonstandard fields, you may need to modify some default <<configure-metrics-source,configuration settings>>.
+To ingest data, you can use {metricbeat} installed on each server you want to monitor, or
+third-party collectors that are configured to ship ESC-compliant data. <<metrics-fields-reference>>
```
Comment (bmorelli25) [inline docs/en/metrics/metrics-installation.asciidoc:9]: I haven't heard of ESC. Did you mean ECS? Also, a missing 'the'
```suggestion
third-party collectors that are configured to ship ECS-compliant data. The <<metrics-fields-reference>>
```

File: docs/en/metrics/metrics-installation.asciidoc [hunk 2]
```diff
@@ -1,100 +1,107 @@
...
-[[install-beats-for-metrics]]
-=== Step 3: Set up and run {metricbeat}
-
-To start collecting metrics data, you need to install {metricbeat} and configure the {metricbeat} modules directly from {kib}.
-
-Alternatively, you can install {metricbeat} and configure the {metricbeat} modules yourself.
-
-[float]
-==== Install {metricbeat} from {kib}
+To get started quickly, you can spin up a deployment of our
```
Comment (bmorelli25) [inline docs/en/metrics/metrics-installation.asciidoc:21]:
```suggestion
To get started quickly, spin up a deployment of our
```

File: docs/en/metrics/metrics-installation.asciidoc [hunk 3]
```diff
@@ -1,100 +1,107 @@
...
+https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}]. The deployment includes
+{es} and {kib}, and is available on AWS, GCP, and Azure.
+https://www.elastic.co/cloud/elasticsearch-service/signup?baymax=docs-body&elektra=docs[Try the {es}
+Service for free].
```
Comment (bmorelli25) [inline docs/en/metrics/metrics-installation.asciidoc:25]: All of these are available as [shared attributes](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc)

```suggestion
{ess-product}[hosted {ess}]. The deployment includes
{es} and {kib}, and is available on AWS, GCP, and Azure.
{ess-trial}[Try {ess} for free].
```

File: docs/en/metrics/metrics-overview.asciidoc [hunk 4]
```diff
@@ -19,7 +19,7 @@ Metrics monitoring requires the following {stack} components.
 distributed storage, search, and analytics engine. {es} can store, search, and analyze large volumes of data in near real-time.
 The {metrics-app} uses {es} to store metrics data in {es} documents which are queried on demand.
 
-*https://www.elastic.co/products/beats[{beats}]* are open source data shippers that you install as agents on your servers to send data to {es}.
+*https://www.elastic.co/beats/metricbeat[{metricbeat}]* is an open source data shipper that you install as an agent on your servers to send data to {es}.
```
Comment (bmorelli25) [inline docs/en/metrics/metrics-overview.asciidoc:22]:
```suggestion
*{metricbeat-ref}[{metricbeat}]* is an open source data shipper that you install as an agent on your servers to send data to {es}.
```

File: docs/en/metrics/metrics-installation.asciidoc (hunk 1 above)
Comment (EamonnTP) [reply docs/en/metrics/metrics-installation.asciidoc:9]: Thanks, I did indeed mean ECS.

File: docs/en/metrics/metrics-installation.asciidoc [hunk 5]
```diff
@@ -1,100 +1,104 @@
 [[install-metrics-monitoring]]
 [role="xpack"]
-== Install Metrics
+== Get started
 
-The easiest way to get started with Elastic Logs is by using our hosted {es} Service on Elastic Cloud.
-The {es} Service is available on both AWS and GCP, and automatically configures {es} and {kib}.
+To use the {metrics-app}, you need {es} for storing and searching your data, and {kib}
+(version 6.5 or later) for visualizing and managing it.
```
Comment (dedemorton) [inline docs/en/metrics/metrics-installation.asciidoc:6]: I think we can drop mention of 6.6 now that we're more than half way through the 7.n release cycle.

File: docs/en/metrics/metrics-installation.asciidoc [hunk 6]
```diff
@@ -1,100 +1,104 @@
...
+To ingest data, you can use {metricbeat} installed on each server you want to monitor, or
+third-party collectors that are configured to ship ECS-compliant data. The <<metrics-fields-reference>>
+provides a list of all fields used in the {metrics-app}. 
 
-[float]
-=== Hosted Elasticsearch Service
-
-Skip installing and managing your own {es} and {kib} instance by using our hosted {es} Service.
-{ess-trial}[Try out the {es} Service for free].
+If your data uses nonstandard fields, you may need to modify some default <<configure-metrics-source,configuration settings>>.
```
Comment (dedemorton) [inline docs/en/metrics/metrics-installation.asciidoc:12]: I don't think users will understand what you mean when you say "if your data uses nonstandard fields". From a user's perspective, at this point, they just have systems that they  want to monitor. Fields don't exist until after they've installed Metricbeat and data starts flowing to Elasticsearch. I think you're talking about custom index pattern/names and custom fields, but that's not clear here.

Comment (EamonnTP) [reply docs/en/metrics/metrics-installation.asciidoc:12]: Yes, this relates to custom index patterns and non-default fields. I moved this paragraph to a note statement in Step 4.

File: docs/en/metrics/metrics-overview.asciidoc [hunk 7]
```diff
@@ -19,7 +19,7 @@ Metrics monitoring requires the following {stack} components.
 distributed storage, search, and analytics engine. {es} can store, search, and analyze large volumes of data in near real-time.
 The {metrics-app} uses {es} to store metrics data in {es} documents which are queried on demand.
 
-*https://www.elastic.co/products/beats[{beats}]* are open source data shippers that you install as agents on your servers to send data to {es}.
+*{metricbeat-ref}[{metricbeat}]* is an open source data shipper that you install as an agent on your servers to send data to {es}.
```
Comment (dedemorton) [inline docs/en/metrics/metrics-overview.asciidoc:22]: I think you might want to revert this change. `{metricbeat-ref}` resolves the path to the Metricbeat reference. But here, I think you want to point to the product page like you do for ES and Kibana.

Comment (EamonnTP) [reply docs/en/metrics/metrics-overview.asciidoc:22]: Will do. Thanks.
